client/embed.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+  var wecoWorkEmbeds = document.querySelectorAll('.weco-work-embed');
```

Comment:
Out of interest, why `var`s not `const`s?

Reply:
In browser - although it is being compiled.